New log file opened at Mon Oct 05 15:33:17 CEST 2015 by 2128@PC-14874
Taking next connection to: 51353
New client attempting to connect on port: 51354-51353
  Address = /139.63.20.189:51353
This client on port 51354 has uuid 277c6332-6d39-4138-9293-453d663b36db configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-277c6332-6d39-4138-9293-453d663b36db
Taking next connection to: 51353
New client attempting to connect on port: 51393-51353
  Address = /139.63.20.189:51353
Handshake exception on socket: 51393
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 51353
New client attempting to connect on port: 51394-51353
  Address = /139.63.20.189:51353
This client on port 51394 has uuid d2aaadfa-483d-4230-a27d-2ee035a2640c configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-277c6332-6d39-4138-9293-453d663b36db, play-fork-run-d2aaadfa-483d-4230-a27d-2ee035a2640c
Taking next connection to: 51353
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 51353
